a8c-files/Image.php

```diff
@@ -0,0 +1,202 @@
+<?php
+
+namespace A8C_Files;
```

Comment:
Should we use a cleaner namespace for this? Maybe something like `Automattic\VIP\Files`?